app/uk/gov/hmrc/thirdpartydeveloperfrontend/connectors/ThirdPartyApplicationConnector.scala

```diff
@@ -150,16 +150,6 @@ abstract class ThirdPartyApplicationConnector(config: ApplicationConfig, metrics
       }
   }
 
-  def updateIpAllowlist(applicationId: ApplicationId, required: Boolean, ipAllowlist: Set[String])(implicit hc: HeaderCarrier): Future[ApplicationUpdateSuccessful] =
-    metrics.record(api) {
-      http.PUT[UpdateIpAllowlistRequest, ErrorOrUnit](s"$serviceBaseUrl/application/${applicationId}/ipAllowlist", UpdateIpAllowlistRequest(required, ipAllowlist))
```

Comment:
Should you remove the request object too - I assume that it's not used any more?

Reply:
Oh yes - well spotted.